docs/source/sysadmin_guide/off_chain_settings.rst

```diff
@@ -0,0 +1,300 @@
+Additional steps specify the peers for this node, change the scheduler type
+(optional), and create a network key.
+
+#. Create the validator configuration file by copying the example file.
+
+   .. code-block:: console
+
+      $ sudo cp /etc/sawtooth/validator.toml.example /etc/sawtooth/validator.toml
+
+#. Use ``sudo`` to edit this file.
+
+   .. code-block:: console
+
+      $ sudo vi /etc/sawtooth/validator.toml
+
+
+#. Change the network settings for the validator.
+
+   a. Locate the ``endpoint`` setting, which specifies validator's external URL.
+
+      Replace the default interface and port (``127.0.0.1:8800``) with the
+      values for your node. You can use either the NAT values or the publicly
+      addressable IP address and port.
+
+      .. code-block:: ini
+
+         endpoint = "tcp://{external_interface}:{port}"
+
+   #. Locate the ``bind`` settings. If necessary, change these values for your
+      system. The default values are:
+
+        .. code-block:: ini
+
+           bind = [
+             "network:tcp://127.0.0.1:8800",
+             "component:tcp://127.0.0.1:4004",
+             "consensus:tcp://127.0.0.1:5050"
+           ]
+
+      - ``network`` specifies where the validator listens for communication from
+        validators on other nodes
+
+      - ``component`` specifies where the validator listens for the REST API
+
+      - ``consensus`` specifies the settings for the consensus proxy
+
+      .. tip::
+
+         If the bind interface doesn't exist, you might see a ZMQ error in the
+         ``sawtooth-validator`` systemd logs when attempting to start the
+         validator, as in this example:
+
+         .. code-block:: console
+
+            Jun 02 14:50:37 ubuntu validator[15461]:   File "/usr/lib/python3.5/threading.py", line 862, in run
+            ...
+            Jun 02 14:50:37 ubuntu validator[15461]:   File "zmq/backend/cython/socket.pyx", line 487, in zmq.backend.cython.socket.Socket.bind (zmq/backend/cython/socket.c:5156)
+            Jun 02 14:50:37 ubuntu validator[15461]:   File "zmq/backend/cython/checkrc.pxd", line 25, in zmq.backend.cython.checkrc._check_rc (zmq/backend/cython/socket.c:7535)
+            Jun 02 14:50:37 ubuntu validator[15461]: zmq.error.ZMQError: No such device
+            Jun 02 14:50:37 ubuntu systemd[1]: sawtooth-validator.service: Main process exited, code=exited, status=1/FAILURE
+            Jun 02 14:50:37 ubuntu systemd[1]: sawtooth-validator.service: Unit entered failed state.
+            Jun 02 14:50:37 ubuntu systemd[1]: sawtooth-validator.service: Failed with result 'exit-code'.
+
+#. Set the peering type and peer list (directly connected nodes) for this
+   Sawtooth node.
+
+   a. Locate the ``peering`` setting, which specifies the type of peering
+      approach the validator should take: static or dynamic. Change this setting
+      to ``dynamic`` unless static peering is required. For more information,
```

Comment:
I am not sure we want to recommend dynamic peering as the default. Raft and PBFT will require static peering. Maybe something like: "The peering type you should use depends on the network use case and consensus algorithm used. For example, public networks using an open-membership consensus algorithm should use dynamic peering while onsortium networks or networks using a fixed-membership consensus algorithm should use static peering."